package.json

```
@@ -34,7 +34,7 @@
   "scripts": {
     "start": "node src/index.js api",
     "test": "npm run test:unit",
-    "test:unit": "jest --testMatch '**/test/unit/**/*.test.js'",
+    "test:xunit": "jest --testMatch '**/test/unit/**/*.test.js'",
```

Shouldn't this look something like this instead ([ref](https://github.com/mojaloop/account-lookup-service/pull/462/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R48))?

```suggestion
    "test:unit": "jest --testMatch '**/test/unit/**/*.test.js'",
    "test:xunit": "npm run test:junit",
```